compiler/modules/nback/scaffold.bal

```diff
@@ -239,6 +240,17 @@ class Scaffold {
         }
     }
 
+    function buildRuntimeFunctionCall(llvm:Builder builder, RuntimeFunction rf, llvm:Value[] args) returns llvm:Value? {
```

Comment:
Don't make this a method

Reply:
fixed (I assume this is to move the function out of scaffold)